x-pack/plugins/beats_management/server/routes/beats/configuration.ts

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { schema } from '@kbn/config-schema';
+import { IRouter } from 'src/core/server';
+import { ConfigurationBlock } from '../../../common/domain_types';
+import { ReturnTypeList } from '../../../common/return_types';
+import { wrapRouteWithSecurity } from '../wrap_route_with_security';
+
+/*
+import Joi from 'joi';
+import { FrameworkRequest } from '../../lib/adapters/framework/adapter_types';
+import { CMServerLibs } from '../../lib/types';
```

Comment:
I kept the legacy route registration commented in the route files to ease the diff when reviewing. I will remove them all before merging